src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -0,0 +1,828 @@
+                onSendMoney(paymentMethod);
+            } else {
+                // validate the amount for distance requests
+                const decimals = CurrencyUtils.getCurrencyDecimals(iouCurrencyCode);
+                if (isDistanceRequest && !isDistanceRequestWithoutRoute && !MoneyRequestUtils.validateAmount(String(iouAmount), decimals)) {
+                    setFormError('common.error.invalidAmount');
+                    return;
+                }
+
+                if (isEditingSplitBill && TransactionUtils.areRequiredFieldsEmpty(transaction)) {
+                    setDidConfirmSplit(true);
+                    setFormError('iou.error.genericSmartscanFailureMessage');
+                    return;
+                }
+
+                setDidConfirm(true);
+                onConfirm(selectedParticipantsMemo);
+            }
+        },
+        [
+            selectedParticipantsMemo,
+            iouCategory,
+            iouType,
+            onSendMoney,
+            iouCurrencyCode,
+            isDistanceRequest,
+            isDistanceRequestWithoutRoute,
+            iouAmount,
+            isEditingSplitBill,
+            transaction,
+            onConfirm,
+        ],
+    );
+
+    const footerContent = useMemo(() => {
+        if (isReadOnly) {
+            return;
+        }
+
+        const shouldShowSettlementButton = iouType === CONST.IOU.TYPE.SEND;
+        const shouldDisableButton = selectedParticipantsMemo.length === 0 || shouldDisplayMerchantError;
+
+        const button = shouldShowSettlementButton ? (
+            <SettlementButton
+                // @ts-expect-error TODO: Remove this once SettlementButton (https://github.com/Expensify/App/issues/25100) is migrated to TypeScript.
+                pressOnEnter
+                isDisabled={shouldDisableButton}
+                onPress={confirm}
+                enablePaymentsRoute={ROUTES.IOU_SEND_ENABLE_PAYMENTS}
+                addBankAccountRoute={bankAccountRoute}
+                addDebitCardRoute={ROUTES.IOU_SEND_ADD_DEBIT_CARD}
+                currency={iouCurrencyCode}
+                policyID={policyID}
+                buttonSize={CONST.DROPDOWN_BUTTON_SIZE.LARGE}
+                kycWallAnchorAlignment={{
+                    horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
+                    vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                }}
+                paymentMethodDropdownAnchorAlignment={{
+                    horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                    vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                }}
+                shouldShowPersonalBankAccountOption
+            />
+        ) : (
+            <ButtonWithDropdownMenu
+                pressOnEnter
+                isDisabled={shouldDisableButton}
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                onPress={(_event, value) => confirm(value)}
```

Comment:
```suggestion
                onPress={(event, value) => confirm(value)}
```

Reply:
I think this underscore was added for this marking this param as not used so not sure if this will change anything since I think we will still need disable eslint for that line